lib/models/post.dart

```diff
@@ -1,32 +1,36 @@
+import 'package:tenant_finder/constants.dart';
 class Post{
-  static const tblPost = 'post';
-  static const colId = 'id';
-  static const colAddress = 'address';
-  static const colSizeOfPlace = 'sizeOfPlace'; 
-  static const colRentCommencementDate = 'rentCommencementDate';
-  static const colRentalPrice = 'rentalPrice';
-  static const colBookingMoney = 'bookingMoney';
-  static const colDescription = 'description';
-
-
 
   int id;
+  // String status;
+  String posted_by;
   String address;
-  String sizeOfPlace;
-  String rentCommencementDate;
-  String rentalPrice;
-  String bookingMoney;
+  String size_of_place;
+  String bedroom_no;
+  String rent_commencement_date;
+  String rental_price;
+  String booking_money;
+  // String rental_type;
+  bool rental_type_family;
+  bool rental_type_bachelor;
   String description;
 
-  Post({this.id, this.address, this.sizeOfPlace, this.rentCommencementDate, this.rentalPrice, this.bookingMoney, this.description});
+      // this.status, this.rental_type, 
+  Post({this.id, this.posted_by, this.address, this.size_of_place, this.bedroom_no, this.rent_commencement_date, this.rental_price, this.booking_money, this.rental_type_family, this.rental_type_bachelor, this.description});
 
   Map<String, dynamic> toMap(){
     var map = <String, dynamic> {
+      // colStatus: status,
```

Comment:
we don't need to remove status from model as we'll need this on post update. and rename this property to isActive it'll make more sense. When a post is created isActive value should be true by default.